### ISSUE TITLE
Add training resources table

### ISSUE THREAD

**Issue by srghosh56** [EVERSE-ResearchSoftware/RSQKit#117]
### Purpose
Add a structured table displaying WP5 training and recognition resources with proper layout and styling matching RSQKit design

### Implementation
- Add training resources data in yaml format
- Create table layout using Liquid templating
- Ensure styling matches existing RSQKit pages

**Comment by srghosh56** [comment]
Created #118 for this

**Comment by lauportell** [comment]
Decide whether they are they going to be included in resources or as a separate thing in the data schema of RSQKit and the linkage with an EVERSE TeSS instance?

**Comment by shoaibsufi** [comment]
The label status:wait - means that https://github.com/EVERSE-ResearchSoftware/RSQKit/issues/127 would have to be resolved and settled before this issue can progress - it's is hoped this will be done by the end of Dec 2024

**Comment by dgarijo** [comment]
Yes, I need a slot to document the schema (we may only need 1 new property, the rest is bioschemas)
